src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  width: $content-width;
+  justify-content: start;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $gap-block;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    position: relative;
+
+    @include square;
+
+    box-sizing: border-box;
+    border: 1px solid black;
+
+    background-color: $background-color-of-blocks;
+
+    cursor: pointer;
+    transition: transform 0.5s, background-color 0.5s;
```

Comment:
Would be nice to create a variable for duration, then we can use it for every transition to make it's duration consistent with other page transitions